.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -6,5 +6,5 @@ contact_links:
     url: https://groups.google.com/group/zaproxy-develop
     about: Please ask and answer questions about ZAP internals here.
   - name: ZAP Security Bug Bounty Program
-    url: https://bugcrowd.com/owaspzap
-    about: Please report security vulnerabilities here.
+    url: mailto:zaproxy-admin@googlegroups.com
```

Comment:
We also need to remove on the main config file and maybe remove this one if the other applies to all repos.

Reply:
I'll update the other file. This repo has more specifics, and actual issue forms.

Reply:
So?